.github/workflows/fix_permissions.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Can you rename this file to something more explicit please. For example: `change_datadir_permissions_to_app.sh`